actionview/lib/action_view/digestor.rb

```diff
@@ -18,11 +18,11 @@ class << self
       # * <tt>name</tt>   - Template name
       # * <tt>finder</tt>  - An instance of <tt>ActionView::LookupContext</tt>
       # * <tt>dependencies</tt>  - An array of dependent views
-      def digest(name:, finder:, dependencies: nil)
+      def digest(name:, format:, finder:, dependencies: nil)
         if dependencies.nil? || dependencies.empty?
-          cache_key = "#{name}.#{finder.rendered_format}"
+          cache_key = "#{name}.#{format}"
         else
-          cache_key = [ name, finder.rendered_format, dependencies ].flatten.compact.join(".")
+          cache_key = [ name, format, dependencies ].flatten.compact.join(".")
```

Comment:
I don't see how the old code would get a `rendered_format` here? Did you already consolidate `rendered_format=` on the finder/lookup_context or am I misremembering?

Reply:
It's pretty confusing, but [the template renderer would set the lookup context](https://github.com/rails/rails/blob/df12a1b2413906ee38a977e3cbb325512c184837/actionview/lib/action_view/renderer/template_renderer.rb#L13).  The partial renderer [could do it too](https://github.com/rails/rails/blob/df12a1b2413906ee38a977e3cbb325512c184837/actionview/lib/action_view/renderer/partial_renderer.rb#L317-L323).

Reply:
Uhhh, right, the both set the same info but from slightly different vantage points. Got it, thanks 👍